### PR TITLE
Fix gateway mock

### PR DESCRIPTION
**Pull request by thanakritTW**
No ticket for that

Changes

 - Because mainline use v0.5 instead of v1, so I updated the postman collections and gateway stub configs
 - Add more requests on linking phase to  the collections
 - change on docker-compose.yaml

### CODE REVIEW

File: gatewayStubConfigurations/imposters/discoveryResponse.ejs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 (request, state, logger) => {
-    logger.info("/v1/care-contexts/on-discover is called");
```
Comment (praveenadayanand) [inline gatewayStubConfigurations/imposters/discoveryResponse.ejs:2]: can we introduce and use variables for versions in these ejs files?

Comment (thanakritTW) [reply gatewayStubConfigurations/imposters/discoveryResponse.ejs:2]: All I can do is to remove duplications on logging as in latest commit, I cannot see easy way to make the variable in main file

File: docker-compose.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ services:
       - rabbitmq
       - otp
   otp:
-    build: src/In.ProjectEKA.OtpService/.
+    image: "ganesan92/otp-service"
```
Comment (davidviktorTW) [inline docker-compose.yml:12]: Is this intentional change? If so, should it be done in this PR?

Comment (thanakritju) [reply docker-compose.yml:12]: Yes, this is intentional. I feel like another PR of only this change, is kind of wasting everyone's time. If this is a feature branch I would say this change is not meant to be here. But this should be change anyway but no one is doing it. So I included this in this fixing branch.

**Pull request merged**